addon/components/resize-detector.js

```diff
@@ -30,10 +30,9 @@ export default Ember.Component.extend({
     if (this.get('isDestroyed') || this.get('isDestroying')) {
       return;
     }
-    let $el = Ember.$(element);
     this.get('on-resize')({
-      width: $el.width(),
-      height: $el.height()
+      width: element.offsetWidth,
+      height: element.offsetHeight
```

Comment:
offsetWidth and offsetHeight are not good substitutes for jquery width and height, especially if the user is watching window or document.

For the most compatible solution, you would have to use parseFloat(window.getComputedStyle(element).width) and that still wouldn't match for the case of window or document, where the calculation by jquery gets really complex.

It is quite likely that users are depending on existing behavior. I'm not sure what to do.

Reply:
Here's the exact equivalent:

https://github.com/nefe/You-Dont-Need-jQuery#2.2